### ISSUE TITLE
Tibetan Tibetan character tables document

### ISSUE THREAD

**Issue by CFynn** [n8willis/opentype-shaping-documents#39]
The paragraph at the end of [this page](https://github.com/n8willis/opentype-shaping-documents/blob/master/character-tables/character-tables-tibetan.md#miscellaneous-character-table) just under the Miscellaneous Character Table which references  shaping using Halant (and blocking this using ZWJ)   appears to be in error as regards to the Tibetan script as the Halant character should not normally be used in Tibetan shaping - since in Tibetan consonant conjuncts are formed using the set of explicit Subjoined Consonants (U+0F90-U+0FBC) without any need of Halant (called Virama in Tibetan block).
In Tibetan _all_ occurrences of the Halant / VIRAMA character (U+0F84) would normally be displayed with the glyph for that character.

Same thing with the whole next paragraph _"Note, however, that the "consonant,Halant" subsequence in the above example may still trigger a half-forms feature....."_   Again, this paragraph doesn't make sense for Tibetan since Halant (VIRAMA) should _never_ be needed to trigger a half-forms feature in Tibetan (since there are those explicit Subjoined Consonants in the Tibetan encoding).

Similarly with the next paragraph which references usage of the zero-width joiner is to prevent the formation of "Reph". In Tibetan RA followed by any subjoined consonant normally shapes as "ra-go" (abbreviated form  of RA in Tibetan (there are a few exceptions which differ dependent on the particular style of Tibetan script). To prevent this shaping behavior in Tibetan   U+0F6A "Fixed form Ra" is normally used. In Tibetan there is no "Reph".

**Comment by n8willis** [comment]
Indeed, there is search a dearth of information online about Tibetan shaping (at least in English) that there is a lot of boilerplate material in the document brought over from other shapers. So thank you for taking a look and pointing out the updates needed! That really does help a lot.

Along those lines, there are a few code points in the Tibetan block where the Unicode spec itself is lacking some detail.

Are you able to comment on the usage (and possible classification issues) of 'Lci rtags' and 'Yang rtags' (U+0F86 & U+0F87)?

Also, there are five code points designated CONSONANT_HEAD that are described as "for transliteration" in the spec.... U+0F88–U+0F8C. Can you shed further light on how these are used in practice (today in particular, if they are still found in contemporary new documents)?

**Comment by n8willis** [comment]
Hi @CFynn -- any insight you can share on those questions in the comment above? I'd like to not leave the issues unsolved.
